Example/Controllers/Extensions/ViewController+MiniApp.swift

```diff
@@ -112,6 +112,10 @@ extension ViewController: MiniAppNavigationDelegate {
         switch error {
         case .metaDataFailure:
             metaDataFailure()
+        case .invalidSignature:
+            self.displayAlert(title: MASDKLocale.localize("miniapp.sdk.ios.error.title"), message: MASDKLocale.localize(.signatureFailed), dismissController: true) { _ in
```

Comment:
Does App Center let you manually send a crash report/log? Maybe we could do that here so we have that extra information from the Demo App?

Reply:
Unfortunately App Center does not have this feature, we could have a dedicated slack channel for this kind of things